### ISSUE TITLE
Button on Posts Categories (settings/taxonomies/category/) is missing discernible text

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75596]
### Quick summary

A button on Posts Categories is missing discernible text and won't have context for screen reader users.
The button element is:
`<span class="taxonomy-manager__icon" role="button" tabindex="0">`

### Steps to reproduce

1. On settings/taxonomies/category/
2. Inspect the button referenced above or use an accessibility checker like axe DevTools chrome extension to scan the page.

### What you expected to happen

All buttons should have discernible text to provide context to users with screen readers and other assistive devices.

### What actually happened

A button for each category on the page doesn't have discernible text. This is a more significant issue the more categories there are. The user won't be able to tell what the button is intended to do or which one they want to click if there's more than one.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by autumnfjeld** [comment]
@youbrokesomething Can you provide a screenshot of the button you are referring to?

**Comment by miksansegundo** [comment]
@autumnfjeld I found it. I'll push a fix soon.

<img width="452" alt="Screenshot 2566-05-15 at 13 21 32" src="https://github.com/Automattic/wp-calypso/assets/1881481/8ab839d1-a329-4f3d-94df-4bd5cbfaa98b">

**Comment by taipeicoder** [comment]
Fixed via #76930
